### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by Wasca** [vojtechmares/terraform-cloudflare-aws-ses-verification#5]
Hi 

I noticed this project is getting the following Deprecation Warning.

```
Warning: Argument is deprecated

with module.verification.cloudflare_record.domain_verification, on .terraform/modules/verification/main.tf line 18, in resource "cloudflare_record" "domain_verification": 

18:   value   = aws_ses_domain_identity.this.verification_token

`value` is deprecated in favour of `content` and will be removed in the next major release.
```

**Comment by vojtechmares** [comment]
Hi, thanks for reporting, I will fix it and tag a new version!

**Comment by vojtechmares** [comment]
Closed by [v0.2.0](https://github.com/vojtechmares/terraform-cloudflare-aws-ses-verification/releases/tag/v0.2.0)

**Comment by Wasca** [comment]
Hi @vojtechmares 

Sorry to bring this one up again, but lines 18, 30, and 37 need to have the word "value" replaced with "content" in main.tf

I checked v0.2.0 and it does not appear to be updated.

**Comment by vojtechmares** [comment]
@Wasca 

🤦 my bad, fixed in [v0.2.1](https://github.com/vojtechmares/terraform-cloudflare-aws-ses-verification/releases/tag/v0.2.1).
